packaging/standalone/src/main/distribution/shell-scripts/bin/Neo4j-Management/Get-Java.ps1

```diff
@@ -82,10 +82,15 @@ Function Get-Java
     $javaVersion = ''
     $javaCMD = ''
     
+    $EnvJavaHome = Get-Neo4jEnv 'JAVA_HOME'
+    $EnvJavaOpts = Get-Neo4jEnv 'JAVA_OPTS'
+    $EnvClassPrefix = Get-Neo4jEnv 'CLASSPATH_PREFIX'
```

Comment:
Prefix is still referenced in;

https://github.com/neo4j/neo4j/blob/3.0/packaging/standalone/src/main/distribution/shell-scripts/bin/neo4j-shell#L95